lib/statsd.js

```diff
@@ -175,10 +228,10 @@ Client.prototype.sendAll = function(stat, value, type, sampleRate, tags, callbac
 
   if(Array.isArray(stat)){
     stat.forEach(function(item){
-      self.send(item, value, type, sampleRate, tags, onSend);
+      self.sendStat(item, value, type, sampleRate, tags, onSend);
```

Comment:
I had to split the existing send method in two, and the part that is applicable to both events and stats is now called send().